com.unity.multiplayer.mlapi/Prototyping/NetworkNavMeshAgent.cs

```diff
@@ -10,6 +10,7 @@ namespace MLAPI.Prototyping
     /// A prototype component for syncing NavMeshAgents
```

Comment:
NavMeshAgents -> NetworkNavMeshAgents, keeps grep clean

Reply:
I think this comment is fine. NetworkNavMeshAgent is a component to sync a regular NavMeshAgent.